src/sagemaker/jumpstart/utils.py

```diff
@@ -881,6 +890,37 @@ def validate_model_id_and_get_type(
     return None
 
 
+def _validate_hub_service_model_id_and_get_type(
+    model_id: Optional[str],
```

Comment:
just curious, what happens if `model_id` is `None`?

Reply:
Thanks, good catch!